prosody/rootfs/defaults/saslauthd.conf

```diff
@@ -4,27 +4,47 @@
 
 {{ if eq $PROSODY_AUTH_TYPE "ldap" }}
 ldap_servers: {{ .Env.LDAP_URL }}
+
 ldap_search_base: {{ .Env.LDAP_BASE }}
-{{ if .Env.LDAP_BINDDN | default "" }}
-ldap_bind_dn: {{ .Env.LDAP_BINDDN }}
-ldap_bind_pw: {{ .Env.LDAP_BINDPW }}
-{{ end }}
-ldap_filter: {{ .Env.LDAP_FILTER | default "uid=%u" }}
+
 ldap_version: {{ .Env.LDAP_VERSION | default "3" }}
 ldap_auth_method: {{ .Env.LDAP_AUTH_METHOD | default "bind" }}
+
+  {{ if .Env.LDAP_BINDDN | default "" }}
+ldap_bind_dn: {{ .Env.LDAP_BINDDN }}
+ldap_bind_pw: {{ .Env.LDAP_BINDPW | default "MISSING PASSWORD" }}
+  {{ end }}
+
+ldap_filter: {{ .Env.LDAP_FILTER | default "uid=%u" }}
+
+ldap_group_attr: {{ .Env.LDAP_GROUP_ATTR | default "uniqueMember" }}
+  {{ if .Env.LDAP_GROUP_DN | default "" }}
```

Comment:
I don't think the default in the check does anything here.

Reply:
Happy to remove, I took note of it being used already (see pin), I think it might be used to prevent unexpected behavior when the variable is undefined?

Reply:
Done